docs/getting-started/3d/services/csw_service.md

```diff
@@ -10,4 +10,21 @@
 | GetRepositoryItem | GET (KVP) |
 | GetDomain | GET (KVP) / POST (XML) / SOAP |
 
+**_How can I see the records??_**
 
+You need to send a post request with xml describing the specific records you want.
+
+Of course, you have to add the authentication stage in order to get answer.
+
+For more details, see: [Authentication](/ogc-protocols/ogc-csw-auth.md)
+
+An example of a post request's body:
+
+![alt text](../images/post_request.png?raw=true)
+
+After sending the request, you will get a response with all the records matched to the filter query. For example:
+
+![alt text](../images/post_response.png?raw=true)
```

Comment:
give it not as image but as text in "collapsable" mode talk to me if you need an example